analyses/chromosomal-instability/01-localization-of-breakpoints.Rmd

```diff
@@ -75,12 +79,19 @@ Here's all the input files we will need:
 
 ```{r}
 chr_file <- file.path(data_dir, "WGS.hg38.strelka2.unpadded.bed")
-metadata_file <- file.path(data_dir, "pbta-histologies.tsv")
 breaks_file <- file.path(output_dir, "breaks_lists.RDS")
 uncallable_file <- file.path(
   "..", "copy_number_consensus_call", "ref",
   "cnv_excluded_regions.bed"
 )
+
+# set up histologies file
+if ( params$base_run == 0 ){
```

Comment:
Can you confirm that this runs as expected if you pass `0` as the `param`? I [can think of other cases](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/e99c13550956c95fa83e9c811515e04a01a8c3d4/analyses/gene-set-enrichment-analysis/02-model-gsea.Rmd#L45) where `params` did not exactly behave as expected.

Reply:
Do you mean within a v21 branch or within v22 branch which should error out?

Reply:
while in this branch (v22 data download), running: 
```bash
root@334c2bfa1312:/home/rstudio/kitematic/scripts# OPENPBTA_BASE_SUBTYPING=0 bash ../analyses/chromosomal-instability/run_breakpoint_analysis.sh

Error: 
 Could not find needed file(s):
../../data/pbta-histologies.tsv
Check your options and set up.
Execution halted
```

This is expected. I will open a new PR with a v21 run from master.

Reply:
Seems the param works if 1 is default, but not 0.

Reply:
I meant did you do functional testing by passing `params = list(base_run = 0)`. I just did locally.